.pylintrc

```diff
@@ -0,0 +1,37 @@
+[MESSAGES CONTROL]
+
+disable=broad-except,
+        too-few-public-methods,
+        parse-error,
+        invalid-name,
+        attribute-defined-outside-init,
+        pointless-statement,
```

Comment:
This is only needed, because `requirements.txt` is linted.
```
requirements.txt:1:0: W0104: Statement seems to have no effect (pointless-statement)
requirements.txt:2:0: W0104: Statement seems to have no effect (pointless-statement)
requirements.txt:3:0: W0104: Statement seems to have no effect (pointless-statement)
requirements.txt:4:0: W0104: Statement seems to have no effect (pointless-statement)
requirements.txt:5:0: W0104: Statement seems to have no effect (pointless-statement)
requirements.txt:6:0: W0104: Statement seems to have no effect (pointless-statement)
```